src/e2e/helpers/mobile/paste.ts

```diff
@@ -1,13 +1,23 @@
 import { Browser } from 'webdriverio'
+import { WindowEm } from '../../../initialize'
+import { HOME_TOKEN } from '../../../constants'
+
+const em = window.em as WindowEm
+
+async function paste(browser: Browser<'async'>, text: string): Promise<void>
+async function paste(browser: Browser<'async'>, pathUnranked: string[], text: string): Promise<void>
 
 /** Import text on given unranked path using exposed testHelpers. */
-const paste = async (browser: Browser<'async'>, unrankedPath: string[], text: string) => {
+async function paste(browser: Browser<'async'>, pathUnranked: string | string[], text?: string): Promise<void> {
```

Comment:
What do you think about below? Isn't it enough?

```ts
async function paste(page: Page, text: string, pathUnranked = [HOME_TOKEN]): Promise<void> {
  // Note: This helper is exposed because copy paste seemed impossible in headless mode. With headless false copy paste with ctrl + v seems to work. ??
  await page.evaluate((_pathUnranked, text) => {
    const testHelpers = em.testHelpers
    testHelpers.importToContext(_pathUnranked, text)
  }, pathUnranked, text)
}
```

Reply:
It would work. I think it’s more intuitive to pass the path first though, since it is short while the text can take up multiple lines. I’m willing to have a slightly more complex definition of it makes the tests flow better.